src/StochOptInterface/StochOptInterface.jl

```diff
@@ -0,0 +1,22 @@
+# TODO This module should be replaced by the package https://github.com/JuliaStochOpt/StochOptInterface.jl
+module StochOptInterface
+
+using DocStringExtensions
```

Comment:
Should we continue to use DocStringExtensions?

Reply:
No we can get rid of it :)

Reply:
What is the reason to stop using it ?

Reply:
I am in a minimalistic mood for a few months ...
I tend to believe that the fewer the dependancies, the easier the maintenance is!